docs/static_site/src/pages/get_started/osx_setup.md

```diff
@@ -110,13 +110,10 @@ the guide in [Math Library Selection](build_from_source#math-library-selection).
 ```bash
 rm -rf build
 mkdir -p build && cd build
-cmake -GNinja -C ../config.cmake ..
-cmake --build . --parallel 8
+cmake -C ../config.cmake ..
```

Comment:
Is -C ../config.cmake required?

Reply:
Yes

Reply:
I think it's reasonable to require specification of the config file. Otherwise users may not understand how to change the config.

Reply:
We have already asked users to `cp config/linux.cmake config.cmake` before this line of instruction. I think it's already very explicit that `config.cmake` will be used to generate the cmake build. This option seems redundant and may cause confusion if users by chance specify a different file other than config.cmake.

Reply:
> This option seems redundant and may cause confusion if users by chance specify a different file other than config.cmake.

Maybe on the contrary, it can be useful to enable users to keep `config_1.cmake` and `config_2.cmake`, providing an easy way to switch between the two configs.

Another reason for being explicit here is that our project's `CMakeLists.txt` may not only be used by MXNet "users", but also by third party projects adding MXNet as a dependency.

This use-case is not yet well supported, but modifying our `CMakeLists.txt` to attempt reading `config.cmake` may break the use-case. 

I suggest to investigate this further and improve in a separate PR.